package.json

```
@@ -489,7 +489,7 @@
         "@azure/arm-resources": "^5.0.0",
         "@azure/ms-rest-js": "^2.2.1",
         "@microsoft/vscode-azext-azureappservice": "^0.6.4",
-        "@microsoft/vscode-azext-azureutils": "^0.3.1",
+        "@microsoft/vscode-azext-azureutils": "^0.3.4",
         "@microsoft/vscode-azext-utils": "^0.3.7",
```

```suggestion
        "@microsoft/vscode-azext-utils": "^0.3.8",
```
+ a fresh `npm install`